src/TuDa.CIMS.Web/Components/ShoppingCartFooter.razor

```diff
@@ -0,0 +1,9 @@
+﻿<!--- Shopping cart footer component -->
+<MudContainer MaxWidth="MaxWidth.Large" Class="d-flex justify-end align-center p-3 rounded-xl"
+              Style="background-color: white; max-height: 80px">
```

Comment:
⛏️ Some of the styles and classes could be achieved by `MudComponents` like the `MaxHeight.`

Reply:
There is no MaxHeight class for Containers. But the height will be the same without any styles or classes because it's depending on the inner context of the container iirc.

Reply:
There is no MaxHeight class for Containers. But the height will be the same without any styles or classes because it's depending on the inner context of the container iirc.